legend-depot-model/src/main/java/org/finos/legend/depot/domain/entity/StoredEntity.java

```diff
@@ -38,6 +40,7 @@ public StoredEntity(@JsonProperty(value = "groupId") String groupId,
     {
         super(groupId, artifactId, versionId);
         this.entity = entity;
+        this.versionedEntity = false;
```

Comment:
you need to set this to true in the StoreVersionedEntity constructor so we can diff between the teo type